packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -164,10 +164,7 @@ const sendPayload = async (
 
   let body;
 
-  /* Zapier id is hardcoded in the DB, we send the raw data for this case  */
-  if (appId === "zapier") {
-    body = getZapierPayload({ ...data, createdAt });
```

Comment:
We still need that for all previous triggers. Otherwise, we are breaking existing zaps for users as we are returning different data

Reply:
Hey this change it's no longer there, probably seeing and outdated version of the PR.